src/sentry/api/event_search.py

```diff
@@ -285,7 +285,8 @@ def get_snuba_query_args(query=None, params=None):
         elif snuba_name == 'message':
             # make message search case insensitive
             kwargs['conditions'].append(
-                [['positionCaseInsensitive', ['message', "'%s'" % (value,)]], '!=', 0]
+                [['positionCaseInsensitive', [
+                    ['coalesce', ['search_message', 'message']], "'%s'" % (value,)]], '!=', 0]
```

Comment:
I guess we'll see how this coalesce() performs. I guess if events have either one or the other of these columns (but not both) defined, then it should be ok? Or will they have both defined?

Reply:
Also can we use format() for string building

Reply:
THinking a bit more about coalesce. Perhaps it makes sense for the snuba API to do this logic instead? Like, have the client just be able to pretend that everything is still called `message` but the snuba service expands the expression to `coalesce(message, search_message)` behind the scenes?

Reply:
I'm open to that. There is also the question of the user experience. I expect `message` from the user perspective to only search the message and for search message to be some sort of internal thing.